packages/react-redux/modules/hooks/index.ts

```diff
@@ -1 +1,3 @@
 export { default as useAdapterReconfiguration } from './use-adapter-reconfiguration';
+export { default as useUpdateFlags } from './use-update-flags';
+export { default as useUpdateStatus } from './use-update-status';
```

Comment:
Is this used somewhere already?

Reply:
Yes they are used in the `configure` component